internal/agent/message.go

```diff
@@ -4,11 +4,14 @@ import (
 	"github.com/charmbracelet/bubbles/list"
 )
 
-type NewAgentItemsMsg []AgentListItem
+type NewAgentItemsMsg struct {
+	Items    []AgentListItem
```

Comment:
Does anything external use this field? We should make it private if not

Reply:
Technically `loader` does - also imports said `agent` package (`v3/internal/agent`) though a constructor with private fields sounds the better option - will adjust it

Reply:
And tweaked - privatised the fields and made a constructor